pkg/terminal/command.go

```diff
@@ -1726,10 +1727,77 @@ func toggle(t *Term, ctx callContext, args string) error {
 }
 
 func breakpoints(t *Term, ctx callContext, args string) error {
-	breakPoints, err := t.client.ListBreakpoints(args == "-a")
+	// Parse arguments
+	var showAll bool
+	var saveFile string
+	
+	if args != "" {
+		argv := strings.Fields(args)
+		for i, arg := range argv {
+			switch arg {
+			case "-a":
+				showAll = true
+			case "-save":
+				if i+1 >= len(argv) {
+					return errors.New("missing filename after -save flag")
+				}
+				saveFile = argv[i+1]
+				break // Exit loop since we found -save and its argument
```

Comment:
This doesn't exit the loop because it's inside a switch, it doesn't do anything. You need to use a label.